src/layout.jl

```diff
@@ -635,36 +630,67 @@ PlaceHolder(ren::AbstractRenderable) = PlaceHolder(ren.measure.w, ren.measure.h)
 #                                     GRID                                     #
 # ---------------------------------------------------------------------------- #
 
+"""
+    calc_nrows_ncols(n, aspect=(16, 9))
+
+Computes the number of rows and columns to fit a given number `n` of subplots in a figure with aspect `aspect`.
+Adapted from: stackoverflow.com/a/43366784
+"""
+function calc_nrows_ncols(n, aspect=(16, 9))
```

Comment:
Might be nice to also allow for the aspect ratio to be a single number (e.g. `1` for a square grid)

Reply:
Done.

Reply:
thanks!